### DISCUSSION TITLE
Editing fields in many to many table list

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#5005] [Archive]
Hi there!

Does anyone have any experience in making a/some fields of a related item editable in the list of the "parent" object?

Cheers!

**Comment by benhaynes** [comment]
You'll want to create a relational field... you can read about the different types here: https://docs.directus.io/concepts/relationships/

**Reply by keesvanbemmel** [reply]
Sorry I think my question wasn't clear.

I would like to edit field values of a relation in the admin ui.

So let's say I have a product with a m2m relation on suppliers. The relation table has "price" as a field.

Now in the product screen, where there is a table listing the related suppliers and their price, I want to be able to edit that price.

Any thoughts?

**Reply by benhaynes** [reply]
Haha, maybe I'm still confused. In our M2M interface, you simply **click** the related item (in this case, you'd click the "Supplier") and that would open a drawer to edit the item. This assumes you have the interface and your permissions configured properly.

**Reply by kees-tce** [reply]
Ah! I understand the confusion, I'm looking for a way to make an edit inline in the table itself :)

**Reply by benhaynes** [reply]
Ahh, then you're looking for a relational repeater interface, which is still on the feature request board. 

https://github.com/directus/directus/discussions/3962

We do have a "Static" Repeater... but that stores data as JSON, not relationally.

**Reply by kees-tce** [reply]
Thanks man!
